salt/heroku/proxy_config.sls

```diff
@@ -0,0 +1,4 @@
+configure_heroku_proxy:
+  salt_proxy.configure_proxy:
+    - proxyname: heroku_proxy
```

Comment:
Don't we need to have a different proxy config for each app?

Reply:
Yeah u're right. Was hoping that one would be able to do it, but after u mentioned it, one surely wouldn't make sense in this case.